frontend/config/agents.ts

```diff
@@ -28,7 +34,14 @@ export const AGENT_CONFIG: {
           [TokenSymbol.XDAI]: 0.1,
         },
         [WalletType.Safe]: {
-          [TokenSymbol.XDAI]: 2,
+          [TokenSymbol.XDAI]: Number(
+            formatEther(
+              `${
+                (traderFundRequirements?.agent ?? 0) +
+                (traderFundRequirements?.safe ?? 0)
+              }`,
+            ),
+          ),
```

Comment:
this will increase this number from 2 to something like 5. Just confirming that after this change, the user will not need to add more funds and it's only the suggested value that's shown while the first setup, right?

Reply:
yes they may need to add more funds, and in this case the operating balance is too low alert should be shown

and it's not the value shown during the initial setup, that value remains 10 xDai. this one is for showing low balance alert